### ISSUE TITLE
Versions of Boost > 1.55 cause failures in tessellation(?)

### ISSUE THREAD

**Issue by pbtoast** [pbtoast/polytope#6]
Reading the INSTALL, I found a note about how newer versions of Boost "cause polytope to fail". Is this still an issue?

**Comment by jmikeowen** [comment]
Yes, dealing with that is why I created the NewQuantTess branch, which is
still incomplete.

**Comment by pbtoast** [comment]
Well, feel free to dump thoughts and observations in this issue if you ever want to tiger-team it.
